build/Microsoft.DotNet.Cli.Run.targets

```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="14.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+
+  <Target Name="CleanArtifacts" >
+    <RemoveDir Directories="$(OutputDirectory)" />
```

Comment:
Nope, it works when the directory has files in it...just ran a full build then used the clean command :)